internal/admission/controller/main.go

```diff
@@ -54,7 +55,9 @@ func (ia *IngressAdmission) HandleAdmission(ar *v1beta1.AdmissionReview) error {
 
 	ingressResource := v1.GroupVersionResource{Group: networking.SchemeGroupVersion.Group, Version: networking.SchemeGroupVersion.Version, Resource: "ingresses"}
 
-	if ar.Request.Resource == ingressResource {
+	oldIngressResource := v1.GroupVersionResource{Group: extensions.SchemeGroupVersion.Group, Version: extensions.SchemeGroupVersion.Version, Resource: "ingresses"}
```

Comment:
@johnlanni this is not enough. The CheckIngress method expects an Ingress from the networking package

Reply:
> @johnlanni this is not enough. The CheckIngress method expects an Ingress from the networking package

I have tested the networking ingress can be deserialized from the request. The ingress protocol is compatibility between networking group and extensions group.